app/templates/dayview.html

```diff
@@ -1,68 +1,96 @@
 <!DOCTYPE html>
```

Comment:
Can you please reformat this file? Some of the old (good) formatting has been ruined

Reply:
fixed